packages/api/src/Concept/Concept.resolver.ts

```diff
@@ -34,6 +34,17 @@ export class ConceptResolver {
     return this.userConceptService.findByUser(user.id);
   }
 
+  @UseGuards(GQLAuthGuard)
+  @Query(() => [UserConcept])
+  async myPathConcepts(
+    @CurrentUser() user: User,
+    @Args('pathId') pathId: string
+  ) {
+    const userConcepts = await this.userConceptService.findByUser(user.id);
+    if (!userConcepts) throw new NotFoundException('UserConcept not found');
```

Comment:
I think this should be handled in the service

Reply:
move to service by adding findByUserByPath service. Commit 4a12935. feat(api): modify according to first two comments